components/SubTrackDetails/SubTrackDetails.jsx

```diff
@@ -0,0 +1,20 @@
+require('./SubTrackDetails.scss')
+
+import React, { Component } from 'react'
+
+class SubTrackDetails extends Component {
```

Comment:
Please use a functional component instead of ES6 class, since this component is stateless. (https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components)